### ISSUE TITLE
update to next.js 12

### ISSUE THREAD

**Issue by ratashaman** [ecoinomist/babylon-next#1]
I try to update all the package to the latest, but I find this error 

`../../node_modules/next/dist/client/next.js
Error: Configuration contains string/RegExp pattern, but no filename was passed to Babel
    at matchPattern`

any suggestion what should I change?
Thanks!

**Comment by ecoinomist** [comment]
Did you remove `core` or `client` monorepos? If so, you need to remove them from `https://github.com/ecoinomist/babylon-next/blob/main/config/index.js`, because the error seems to come from `https://github.com/ecoinomist/babylon-next/blob/main/repos/web/babel.config.js`

**Comment by ratashaman** [comment]
nope, I still use those folder, I guess there's something to do with `next.config.js` or `next-transpile-modules` perhaps?

**Comment by ecoinomist** [comment]
yes, if I remember correctly, from Next 12, `next-transpile-modules` should not be used, or need to change config, because Next opts for [import modules](https://nextjs.org/blog/next-12#es-modules-support-and-url-imports), instead.
